README.md

```diff
@@ -106,6 +109,23 @@ $2Behavior = [$2BehaviorImpl];
 <script src="${0:bower_components}/webcomponentsjs/webcomponents-lite.js"></script>
 ```
 
+### [wcs] load web components polyfill when not supported
+
+```html
+<!-- Load Web Components when not supported -->
+<script>
```

Comment:
Ok, I should have known this :smile: [since](https://youtu.be/g7f1Az5fxgU?t=9m46s)